app/models/source.rb

```diff
@@ -0,0 +1,8 @@
+class Source < ActiveRecord::Base
+  belongs_to :source_set
+  validates :aggregation, presence: true
+
+  def aggregation_uri
+    'http://dp.la/item/' + aggregation
```

Comment:
Since PR#12 has been merged, I rebased to develop used the `config` gem to make this URI configurable.